### DISCUSSION TITLE
Why not add collabration scratch addons?

### DISCUSSION THREAD

**Discussion by rticked** [ScratchAddons/ScratchAddons#7788] [Ideas] [answered]
If we add collabration it will be a great idea more people will install the addon etc

**Comment by rgantzos** [comment]
Because it could be used as an unmoderated chat and it would require a server, which Scratch Addons does not have.

**Reply by rticked** [reply]
can u fix blocklive pls

**Reply by rticked** [reply]
Ty for answer and fixing blocklive

**Reply by CST1229** [reply]
we do not own blocklive, and also blocklive has some... pretty suspicious stuff (basically the creator has once used it as a follow bot, and it has privacy issues)

**Reply by rticked** [reply]
@CST1229  why not make a collab adddon it will be more secure also

**Comment by DNin01** [comment, marked as answer]
It would probably have to be limited to local network collaboration.

**Reply by rticked** [reply]
yea

**Reply by rticked** [reply]
ig ill close this disscosion ty guys
